tests/unit/test_serialization.py

```diff
@@ -69,11 +70,21 @@ async def test_deserialization_fails(default_config_path, mock_registry_client):
     schema["name"] = "BadUser"
     schema["fields"][0]["type"] = "int"
     obj = {"name": 100, "favorite_number": 2, "favorite_color": "bar"}
-    writer = avro.io.DatumWriter(avro.io.schema.parse(json.dumps(schema)))
+    writer = avro.io.DatumWriter(avro.schema.parse(json.dumps(schema)))
```

Comment:
I wonder if there is a way of creating the schema from a dict, this seems silly at a glance.

Reply:
Sure, `avro.schema.make_avsc_object`.